asserts/snapasserts/validation_sets.go

```diff
@@ -467,10 +467,16 @@ func (v *ValidationSets) constraintsForSnap(snapRef naming.SnapRef) *snapContrai
 // presence of the given snap as required and the required revision (or
 // snap.R(0) if no specific revision is required).
 // The method assumes that validation sets are not in conflict.
-func (v *ValidationSets) CheckPresenceRequired(snapRef naming.SnapRef) ([]string, snap.Revision) {
+func (v *ValidationSets) CheckPresenceRequired(snapRef naming.SnapRef) ([]string, snap.Revision, error) {
```

Comment:
we probably want to document the error in the doc comment

Reply:
Done.